apps/extension/src/app/pages/home/home.tsx

```diff
@@ -1,9 +1,82 @@
-import { useFlags } from '@app/features/feature-flags';
+import { Route } from 'react-router';
 
-import { HomeV1 } from './home-v1';
-import { HomeV2 } from './home-v2';
+import { HomePageSelectors } from '@tests/selectors/home.selectors';
+import { Box, Stack } from 'leather-styles/jsx';
+
+import { RouteUrls } from '@shared/route-urls';
+
+import { formatCurrency } from '@app/common/currency-formatter';
+import { whenPageMode } from '@app/common/utils';
+import { ActivityList } from '@app/features/activity-list/activity-list';
+import { Collectibles } from '@app/features/collectibles/collectibles';
+import { FeedbackButton } from '@app/features/feedback-button/feedback-button';
+import { PromoBanner } from '@app/features/promo-banner/promo-banner';
+import { Assets } from '@app/pages/home/components/assets';
+import { homePageModalRoutes } from '@app/routes/app-routes';
+import { ModalBackgroundWrapper } from '@app/routes/components/modal-background-wrapper';
+import { AccountCard } from '@app/ui/components/account/account.card';
+
+import { AccountActions } from './components/account-actions';
+import { HomeTabs } from './components/home-tabs';
+import { useHomePageState } from './use-home-page-state';
+
+const emptyBalanceDisplay = '-.--';
 
 export function Home() {
-  const { extension_revamp } = useFlags();
-  return extension_revamp ? <HomeV2 /> : <HomeV1 />;
+  const {
+    balance,
+    isFetchingBnsName,
+    isPrivateMode,
+    name,
+    togglePrivateMode,
+    toggleSwitchAccount,
+  } = useHomePageState();
+
+  return (
+    <Stack
+      data-testid={HomePageSelectors.HomePageContainer}
+      px={{ base: 0, md: 'space.05' }}
+      py={{ base: 0, md: 'space.07' }}
+      gap={{ base: 0, md: 'space.06' }}
```

Comment:
Pretty please use array notation for responsive styles, we use this everywhere else in extension

A case why, beyond consistency:

1. I don't remember all the breakpoints by heart, and I don't want to. Is the next increment up xs or sm? With array notation, there is no ambiguity. The second breakpoint is in the second index of the array.
2. Ordering. Nothing stops someone mixing the order of these properties. Breakpoint styles are linear in their nature, smallest to biggest. When order matters, an array is usually the best choice.